.github/workflows/validate-revision.yml

```diff
@@ -0,0 +1,344 @@
+name: Validate Revision
+
+# It's almost exact copy of validate.yml and has to be maintainted in concert with it.
+# The difference is in the `on` attribute and the step where we do >> cabal.project.validate
```

Comment:
Could we then have an “☞ N.B. If you modify this, incorporate the same modifications in `validate-revision.yml`!” in `validate.yml`?

Reply:
Yeah, good point. I'm thinking now if there may be a way to reuse the existing workflow, so that we avoid duplication altogether. I'll keep you posted!